client/my-sites/exporter/option-fieldset.jsx

```diff
@@ -36,6 +36,29 @@ module.exports = React.createClass( {
 		};
 	},
 
+	renderMenu( menu, key ) {
+		if ( this.props.isLoadingOptions ) {
+			return (
+				<div key={ key } className="exporter__placeholder-text">
+					Loading options...
```

Comment:
Good point. This one is actually made invisible with CSS (it displays as the gray placeholder box), but screen-readers would use it